docs/articles/user_acquisition_analytics.md

```diff
@@ -0,0 +1,442 @@
+Also, we can derive our NumberSpace min and max from this distribution.
+
+Now let's examine the distribution of new users per signup date.
+
+```python
+dates: pd.DataFrame = pd.DataFrame(
+    {"date": [str(datetime.fromtimestamp(ts).date()) for ts in user_df["signup_date"]]}
+)
+dates_to_plot = pd.DataFrame(dates.value_counts(), columns=["count"]).reset_index()
+alt.Chart(dates_to_plot).mark_bar().encode(
+    alt.X("date", title="Signup date"), alt.Y("count", title="Number of subscribers")
+).properties(height=400, width=1200)
+```
+
+![new users per signup date](../assets/use_cases/user_acquisition_analytics/new_users-signup_date.png)
+
+[observations] First, the distribution confirms that our second campaign (December) works much better than the first (August). Second, the jump in signups at 2023-12-21 is due to the second campaign (our data is exclusively campaign-related). To analyze the two campaigns, we need two periods: a first period of 65 days and a second of 185 days seems appropriate.
+Of our 8k users, roughly 2k subscribed in the first campaign period (65 days), and 6k in the second. Since we already know which ad_creatives users clicked on to go sign up (2855 on old campaign ads, 5145 on new campaign ads), we know that some of the roughly 6k users who signed up after the second (December) campaign started, clicked on old (August) campaign ads that occurred *after* the new campaign ad began, possibly after seeing the new campaign ads. [We'll see how this breaks out below.]
+
+so far.. we know that:
+
+- many more users signed up in response to the influencer-backed ad creatives (second campaign) than the first campaign
+- the vast majority of users are low activity, and more recent subscribers
+
+We don't know....
+
+- what kind of ad creatives have similar meanings and possibly outcomes as a result
+- what kinds of users (in terms of activity level) responded to what kind of ad_creatives (in terms of semantic meaning)
+
+We can use embedding and Superlinked spaces to help reveal answers to our remaining questions..
+
+
+## Embedding with Superlinked
+
+Now, let's use Superlinked to embed our data in a semantic space - to:
+ 
+1. inform the model re *which ad creatives* generated **signups**, and *which users* signed up... 
+2. group ad creatives that have similar meanings..
+
+Define a schema for our user data:
+
+```python
+class UserSchema(Schema):
+    ad_creative: String
+    activity: Float
+    signup_date: Timestamp
+    id: IdField
+```
+
+```python
+user = UserSchema()
+```
+
+Now we create a semantic space for our ad_creatives using a text similarity model. Then encode user activity into a numerical space to represent users' activity level. We also encode the signup date into a recency space, allowing our clustering algorithm to take account of the two specific periods of signup activity (following our two campaign start dates).
+
```

Comment:
@morkapronczay
may seem obvious, but so I understand: 
we are using TextSimilaritySpace, NumberSpace, and RecencySpace to arrive at our user clusters, right?

Reply:
the embeddings, or spaces yes produce vectors which are clustered.